mobile/src/store/uam/actions.ts

```diff
@@ -27,12 +28,20 @@ const deleteGroup = createAsyncThunk<
   number,
   GroupsDeleteRequestParamDto,
   AsyncThunkConfig
->(ActionType.DELETE_GROUP, async (payload, { extra }) => {
+>(ActionType.DELETE_GROUP, async (payload, { extra, getState, dispatch }) => {
+  const {
+    auth: { user: currentUser },
+  } = getState();
   const { groupsApi } = extra;
-
+  const { id } = payload;
+  const groupInfo = await groupsApi.getById(payload);
   await groupsApi.delete(payload);
 
-  const { id } = payload;
+  if (currentUser) {
+    if (groupInfo.userIds.includes(currentUser?.id)) {
```

Comment:
It's better to avoid nested if conditions
```suggestion
  if (currentUser && groupInfo.userIds.includes(currentUser.id)) {
```